ReactWindows/ReactNative.Shared/UIManager/BaseViewManager.cs

```diff
@@ -16,7 +24,8 @@ namespace ReactNative.UIManager
     /// </summary>
     /// <typeparam name="TFrameworkElement">Type of framework element.</typeparam>
     /// <typeparam name="TLayoutShadowNode">Type of shadow node.</typeparam>
-    public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> : ViewManager<TFrameworkElement, TLayoutShadowNode>
+    public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> :
```

Comment:
>BaseViewManager [](start = 26, length = 15)

I actually think it might be better to keep all the view managers separate.  They are separate UI frameworks, so why not completely diverge the UI implementations?  Thoughts @matthargett?